WTG.Analyzers.TestFramework/paket.references

```diff
@@ -1,2 +1,5 @@
-﻿Microsoft.CodeAnalysis.CSharp.Workspaces
-NUnit
+group Test
```

Comment:
I would rather leave WTG.Analyzers.TestFramework on 2.0 for now, updating it might pose a problem for the CW1 analyzers.

Reply:
Can't do this, we need it to reference `LanguageVersion.CSharp_7_3`